src/d3a/models/market/two_sided_pay_as_bid.py

```diff
@@ -48,12 +48,11 @@ def __repr__(self):  # pragma: no cover
                     self.accumulated_trade_price
                     )
 
-    def bid(self, price: float, energy: float, buyer: str, seller: str, bid_id: str=None,
-            iaa_fee: bool = False) -> Bid:
+    def bid(self, price: float, energy: float, buyer: str, seller: str, bid_id: str=None) -> Bid:
         if energy <= 0:
             raise InvalidBid()
-        if iaa_fee:
-            price = price * (1 - self.transfer_fee_ratio) + self.transfer_fee_const * energy
+
+        price = price * (1 - self.transfer_fee_ratio) - self.transfer_fee_const * energy
```

Comment:
This change is correct and should have been implemented in my PR when adding the transfer_fee_const. 
IMO you should also change the adding of the constant grid fee in https://github.com/gridsingularity/d3a/blob/b8dc079dda704781b88e6d5a9a1b14ba72b17e7e/src/d3a/models/market/two_sided_pay_as_bid.py#L90 . Right?